### PR TITLE
Release v1.1.3

### PR DESCRIPTION
**Pull request by vashjs**

### CODE REVIEW

**Review by ncovercash — APPROVED** [review]

**Review by ncovercash — CHANGES_REQUESTED** [review]
Can this wait for my latest PR? #115 needs a second review then it should be good.

**Comment by vashjs** [comment]
> Can this wait for my latest PR? #115 needs a second review then it should be good.

Hi @ncovercash, It's already released and this PR just update master with changelog and package.json. Do you want me to wait untill you will release 1.1.4 and we can merge it in scope of this PR as well?

**Review by ncovercash — APPROVED** [review]

**Comment by ncovercash** [comment]
oops, didn't realize you were asking me a question @vashjs . Let's go ahead and merge this

**Pull request merged**
